core/spec/models/spree/shipment_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'spec_helper'
 require 'benchmark'
 
-describe Spree::Shipment, :type => :model do
-  let(:stock_location) { create(:stock_location) }
+describe Spree::Shipment, type: :model do
+  subject { shipment }
```

Comment:
I added subject but need to clean it up replacing all the shipment calls.
Will do that in the morning if someone else doesn't beat me to it :D

On Thursday, 24 December 2015, Andrew Thal <notifications@github.com> wrote:

> In core/spec/models/spree/shipment_spec.rb
> <https://github.com/solidusio/solidus/pull/613#discussion_r48372446>:
>
> > @@ -1,8 +1,8 @@
> >  require 'spec_helper'
> >  require 'benchmark'
> >
> > -describe Spree::Shipment, :type => :model do
> > -  let(:stock_location) { create(:stock_location) }
> > +describe Spree::Shipment, type: :model do
> > +  subject { shipment }
>
> are we using subject somewhere? i am not able to find where
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solidusio/solidus/pull/613/files#r48372446>.
>


-- 
--
Mike's on the move!